environment/stats/src/cloud/benchflow/collectors/stats.go

```diff
@@ -105,6 +104,50 @@ func collectStats(container Container) {
 	}()
 }
 
+func collectNetworkStats(container Container, client docker.Client) {
+	go func() {
+		foNet, err := os.Create("/app/"+container.Name+"_network_tmp")
+		foTop, err := os.Create("/app/"+container.Name+"_top_tmp")
+		if err != nil {
+	        panic(err)
+	    }
+		interfaces, err := net.Interfaces()
```

Comment:
Why don't you run the following two blocks of code (i.e., `nethogs` and `docker top`) in two different `goroutines`?